### PR TITLE
Next batch of changes for v1.0

### PR DESCRIPTION
**Pull request by dvandyk**

### CODE REVIEW

File: python/eos/plot/plotter.py [hunk 1]
```diff
@@ -1300,6 +1300,59 @@ def plot(self):
...
+
+        The following keys are optional:
+
+         * ``bandwidth`` (*float*) -- The factor by which the automatically determined kernel bandwidth is scaled. See the SciPy documentation
+           for ``gaussian_kde``, ``bw_method='silverman'``. Defaults to 1.
+         * ``levels`` (a *list* containing ``'lines'``, ``'areas'``, or both) -- The setting for the illustration of the contours.
+           If ``'lines'`` is provided, the contour lines are drawn.
+           If ``'areas'`` is provided, the contour areas are filled.
+           Defaults to ``['lines']``.
+         * ``contours`` (*list* of *float*) -- The probability levels of the contours. Defaults to ``[0.68, 0.95, 0.99]``.
```
Comment (pluegh) [inline python/eos/plot/plotter.py:1331]: Defaults to ``[0, 0.68, 0.95, 0.99]``

File: python/eos/plot/plotter.py [hunk 2]
```diff
@@ -1300,6 +1300,59 @@ def plot(self):
...
+        kernel density estimate (a 2D smooth histogram) of samples of a probability density,
+        be it a prior, a posterior, or a signal PDF.
+
+        The following key is mandatory:
+
+         * ``data`` (*dict*, see below) -- The data on the probability density that will be histogramed.
+
+           Within the data object, the following keys are understood.
+
+            * ``samples`` (*list* of *float*) -- The samples that will be histogramed. Mandatory.
```
Comment (pluegh) [inline python/eos/plot/plotter.py:1318]: *list* of *float* with shape (N, 2)

File: python/eos/plot/plotter.py [hunk 3]
```diff
@@ -1522,6 +1575,55 @@ def plot(self):
...
+
+        Additional optional keys are:
+
+         * ``bins`` (*int*, optional) -- The number of bins in each dimension. Defaults to 100.
+
+        Example:
+
+        .. code-block::
+
+           analysis = ... # eos.Analysis object as discussed in the example notebook `inference.ipynb`
```
Comment (pluegh) [inline python/eos/plot/plotter.py:1603]: In `simulation`, not `inference`, I think.

File: python/eos/plot/plotter.py (hunk 2 above)
Comment (dvandyk) [reply python/eos/plot/plotter.py:1318]: Fixed.

**Pull request merged**